### ISSUE TITLE
Adding css files to unglue css section

### ISSUE THREAD

**Issue by nadar** [unglue-workflow/server#3]
it should be possible to also include a css file into the current css stack:

like:

```js
"css" : [
        "scss/test.css",
        "scss/example.scss"
],
```

If the css file is add the api responses with `Error: An error occured during the sass render process: File to read not found or unreadable: /var/www/example/app/scss/test.css`

**Comment by TheMaaarc** [comment]
I see one Problem. With the the config you described above, you would send 2 requests.
The first one:
```
mainFile: "scss/test.css"
files: [...]
```
And then the second one:
```
mainFile: "scss/example.scss"
files: [...]
```

Which would then generate 2 files in the end.

But the desired behaviour would be to have one compiled file, right?
That means you will have to send me multiple `mainFiles`, so I can process the `.css` and `.scss` files and compile them into one.

**Comment by nadar** [comment]
this is what currently already happens already, if you have multiple scss files in the unglue file it generates a request for every scss/css file and parsers all responses into $code and generate one file from all the responses:

https://github.com/unglue-workflow/client/blob/master/src/tasks/CssFileHandler.php#L45-L62

**Comment by TheMaaarc** [comment]
Resolved in server version 1.0.16 and version 1.0.0 of the [client](https://github.com/unglue-workflow/client).
